docs/source/reservations/reservations.rst

```diff
@@ -0,0 +1,182 @@
+
+.. _reservation:
+
+Reservations & DWS
```

Comment:
Why not simply Reservations / Reserved instances?

Reply:
DWS is not exactly a reservation, but more like a scheduler, as you don't know when the job will start in DWS. Also, I don't think many people think DWS as a reservation. Calling it `Reservation Instance` may make people miss the support of DWS. Another way to call this page is `Get High-End GPUs`. Wdyt?

Reply:
`Reserved, Capacity Blocks, DWS`?